src/theme/foundations/typography.ts

```diff
@@ -27,10 +27,6 @@ export const typography = {
   },
 
   fontWeights: {
-    hairline: 100,
-    thin: 200,
-    light: 300,
-    normal: 400,
```

Comment:
Why are you getting rid of these?

Reply:
@Simplypheyie96 wanted the minimum font weight to be 500

Reply:
Okay...but we don't need to touch these.